tests/framework/e2e/cluster.go

```diff
@@ -509,27 +509,27 @@ func (cfg *EtcdProcessClusterConfig) EtcdServerProcessConfig(tb testing.TB, i in
 	}
 
 	args := []string{
-		"--name", name,
-		"--listen-client-urls", strings.Join(curls, ","),
-		"--advertise-client-urls", strings.Join(curls, ","),
-		"--listen-peer-urls", peerListenUrl.String(),
-		"--initial-advertise-peer-urls", peerAdvertiseUrl.String(),
-		"--initial-cluster-token", cfg.ServerConfig.InitialClusterToken,
-		"--data-dir", dataDirPath,
-		"--snapshot-count", fmt.Sprintf("%d", cfg.ServerConfig.SnapshotCount),
+		"--name=" + name,
```

Comment:
a quick question: why change this? do you see any issue?

Reply:
it seems that you just want to make it easier to verify the args in UT?

Reply:
Yes, I want to be 100% sure that switching to dynamically generated flags doesn't make any changes in the provided flags.